package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Cree Intelligent Dictionary",
   "scripts": {
     "start": "rollup --config --watch & pipenv run dev",
-    "bg-watch": "rollup --watch &",
+    "bg-watch": "rollup -c && rollup --watch &",
```

Comment:
Interesting! Why this change?

Reply:
I think `rollup --watch` does not build anything when you start the command. I may be wrong but adding rollup -c has no impact